src/components/Observations/ObservationsHome.tsx

```diff
@@ -0,0 +1,64 @@
+import { useCallback, useEffect, useState } from 'react';
```

Comment:
i extracted this out from index.tsx (the request dispatching is now handled by index.tsx.. it eventually instantiates the component needed for the view - for now we have just the main/home view which shows the list vs map components)